src/features/accessibilite/pages/accessibilite/accessibilite.page.html

```diff
@@ -0,0 +1,98 @@
+<div class="container m-5">
+  <h1 class="my-5 mx-5">Déclaration d'accessibilité</h1>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        <strong>Agence Nationale de la Cohésion des Territoires</strong>
+        s’engage à rendre son service accessible, conformément à l’article 47 de la loi n° 2005-102 du 11 février 2005.
+      </p>
+      <p>
+        Cette déclaration d’accessibilité s’applique à
+        <strong>Cartographie Nationale.</strong>
+      </p>
+    </div>
+  </div>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">État de conformité</h3>
+  <p class="mx-5">
+    <strong>Cartographie Nationale</strong>
+    est non conforme avec le RGAA 4.1. Le site n'a encore pas été audité.
+  </p>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">Établissement de cette déclaration d'accessibilité</h3>
+  <p class="mx-5">
+    Cette déclaration a été établie le
+    <strong>21 novembre 2022.</strong>
+  </p>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">Amélioration et contact</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        Si vous n’arrivez pas à accéder à un contenu ou à un service, vous pouvez contacter le responsable de
+        <strong>Cartographie Nationale</strong>
+        pour être orienté vers une alternative accessible ou obtenir le contenu sous une autre forme.
+      </p>
+      <ul>
+        <li>
+          E-mail :
+          <a href="mailto:cartographie.sonum@anct.gouv.fr">cartographie.sonum@anct.gouv.fr</a>
+        </li>
+      </ul>
+      <p>
+        Nous essayons de répondre dans les
+        <strong>2 jours ouvrés.</strong>
+      </p>
+    </div>
+  </div>
+</div>
+<div class="container m-5">
+  <h3 class="my-5 mx-5 text-primary">Voie de recours</h3>
+  <div class="row">
+    <div class="col-7 mx-5">
+      <p>
+        Cette procédure est à utiliser dans le cas suivant : vous avez signalé au responsable du site internet un défaut
+        d’accessibilité qui vous empêche d’accéder à un contenu ou à un des services du portail et vous n’avez pas obtenu de
+        réponse satisfaisante.
+      </p>
+      <p class="my-2">Vous pouvez:</p>
+      <ul>
+        <li>
+          Écrire un message au
+          <a href="https://formulaire.defenseurdesdroits.fr/code/afficher.php?ETAPE=accueil_2016" target="_blank">
```

Comment:
```suggestion
          <a href="https://formulaire.defenseurdesdroits.fr/code/afficher.php?ETAPE=accueil_2016" target="_blank" rel="noopener noreferrer">
```
https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener